auth/src/main/java/com/firebase/ui/auth/ui/email/EmailActivity.java

```diff
@@ -123,7 +124,10 @@ public void onNewUser(User user) {
             FragmentTransaction ft = getSupportFragmentManager().beginTransaction()
                     .replace(R.id.fragment_register_email, fragment, RegisterEmailFragment.TAG);
 
-            if (emailLayout != null) ft.addSharedElement(emailLayout, "email_field");
+            if (emailLayout != null) {
```

Comment:
What about extracting '"email_field"' to local value as this hardcoded string was used twice

Reply:
Turns out it should never have been a local value as it is used in xml as well.